plugins/kafka/src/api/types.ts

```diff
@@ -36,3 +41,11 @@ export interface KafkaApi {
     consumerGroup: string,
   ): Promise<ConsumerGroupOffsetsResponse>;
 }
+
+export interface KafkaDashboardApi {
+  getDashboardUrl(
+    clusterId: string,
+    consumerGroup: string,
+    entity: Entity,
+  ): string | undefined;
```

Comment:
Can we wrap up this response in an `{ url: string? }` instead. It makes it a little easier to grow the API without breaking changes moving forward.

Reply:
implemented change.